kubernetes/dockerfiles/crail-apache/datanode.py

```diff
@@ -0,0 +1,84 @@
+##############  Pocket resource utilization daemon  ####################
+
+#from jsonsocket import Client
+import time
+import psutil
+import ifcfg
+import threading
+import pickle
+import sys
+import socket
+import struct
+
+CONTROLLER_IP = "10.1.47.178"

```

Comment:
This seems to be a hard-coded IP to me. Why?

Reply:
Hi i have removed this python as my modules are not using it and with that the hard-coded link too.
Kindly review my changes and suggest me any further changes if any.